app.js

```diff
@@ -0,0 +1,160 @@
+// HTML elements
+const statusDiv = document.querySelector('.status');
+const resetDiv = document.querySelector('.reset');
+const cellDivs = document.querySelectorAll('.game-cell');
+let player1 = 'player 1';
+let player2 = 'player 2';
```

Comment:
Please reduce global variables this could have been a factory function or a reusable module.
![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png)